packages/web-components/src/components/ic-side-navigation/ic-side-navigation.tsx

```diff
@@ -731,7 +741,9 @@ export class SideNavigation {
       >
         {isSDevice && this.renderTopBar({ ...topBarProps })}
         <div class="side-navigation" id="side-navigation">
-          {!isSDevice && this.renderTopBar({ ...topBarProps })}
+          {!isSDevice &&
+            !this.disableTopBarBehaviour &&
```

Comment:
I think this doesn't need to be added because of lines 709 and 710, as you're already basing the value of `isSDevice` on whether `disableTopBarBehaviour` is true or not. If I've worked out the logic correctly!